packages/lib/src/components/PayNow/PayNow.test.ts

```diff
@@ -0,0 +1,25 @@
+import PayNow from './PayNow';
+
+describe('formatProps', () => {});
```

Comment:
I meant about this describe block 'formatProps' - there is no point to have it in this test, it is not doing anything